src/index.html

```diff
@@ -22,6 +22,357 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo-container">
+        <a
+          class="header__logo"
+          href="#"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link header__link--is-active is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__info">
+          <img
+            class="product-card__image"
+            src="images/imac.jpeg"
+            alt="product"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+        </div>
+        <div class="product-card__reviews">
+          <div class="product-card__reviews-stars">
+            <div class="product-card__star product-card__star--1--filled"></div>
```

Comment:
The classes for stars like 'product-card__star--1--filled' seem overly specific. Consider using a more general approach like 'product-card__star--filled' and apply it to as many stars as needed based on the product rating.